### DISCUSSION TITLE
`Flow` + serialization

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [russhwolf/multiplatform-settings#140] [Q&A]
Is it possible to combine both?

**Comment by ddekanski** [comment]
@russhwolf How come a question asked two years ago didn't get a simple answer, even if it's a "no"?

**Comment by russhwolf** [comment]
Sorry, I guess I missed this question. This is not currently possible but it's tracked in #72

**Reply by ddekanski** [reply]
Thanks for the swift answer and a link to the tracked proposal.
